rules/linux/macos_local_account.yml

```diff
@@ -0,0 +1,46 @@
+title: Local System Accounts Discovery
+id: ddf36b67-e872-4507-ab2e-46bda21b842c
+status: experimental
+description: Detects enumeration of local systeam accounts on MacOS
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/08
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1087.001/T1087.001.md
+logsource:
+  category: process_creation
+  product: macos
```

Comment:
@yugoslavskiy this is something I didnt have 100% sure that we can reference a product named `macos`? I could not find any reference on within the /tools/config folder

Reply:
You are right, there is no such service yet, but the only reason for it is that there was no macos rules (: so you are kinda inventing it right now. I believe the converter will be updated shortly. Thank you!